discord/iterators.py

```diff
@@ -182,7 +182,7 @@ class HistoryIterator(_AsyncIterator):
     `after` parameter to the newest message received. If messages are not
     reversed, they will be out of order (99-0, 199-100, so on)
 
-    A note that if both before and after are specified, before is ignored by the
+    A note that if both `before` and `after` are specified, `before` is ignored by the
```

Comment:
This should be double base ticked right?

Reply:
Fixed in the latest commit